Formula/s/setconf.rb

```diff
@@ -0,0 +1,20 @@
+class Setconf < Formula
+  desc "Utility for easily changing settings in configuration files"
+  homepage "https://setconf.roboticoverlords.org/"
+  url "https://setconf.roboticoverlords.org/setconf-0.7.7.tar.xz"
+  sha256 "19315574540b3181fec31a4059b9e058381e0192317f153d181e7e7e2aa84d86"
+  license "GPL-2.0-or-later"
+  head "https://github.com/xyproto/setconf.git", branch: "main"
+
+  depends_on "python@3.12"
```

Comment:
you can also try out `uses_from_macos "python"`

along with 

`rewrite_shebang detected_python_shebang(use_python_from_path: true), "setconf.py"`

Reply:
Great tip, thanks!

`setconf.py` supports both Python 2 and Python 3, does that change anything?

Reply:
nope, i think we (as well as apple) have dropped py2 support

Reply:
this is just convenient way of not forcing users to use latest py3